modules/nodes-nodes-resources-configuring-auto.adoc

```diff
@@ -0,0 +1,100 @@
+Perform one of the following steps:
+
+.. View the machine config pool:
++
+[source,terminal]
+----
+$ oc describe machineconfigpool <name>
+----
++
+For example:
++
+[source,terminal]
+----
+$ oc describe machineconfigpool worker
+----
++
+.Example output
+[source,yaml]
+----
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfigPool
+metadata:
+  creationTimestamp: 2019-02-08T14:52:39Z
+  generation: 1
+  labels:
+    pools.operator.machineconfiguration.openshift.io/worker: "" <1>
+----
+<1> If a label has been added it appears under `labels`.
+
+.. If the label is not present, add a key/value pair:
++
+[source,terminal]
+----
+$ oc label machineconfigpool worker custom-kubelet=small-pods
+----
+
+.Procedure
+
+. Create a custom resource (CR) for your configuration change:
++
+.Sample configuration for a resource allocation CR
+[source,yaml]
+----
+apiVersion: machineconfiguration.openshift.io/v1
+kind: KubeletConfig
+metadata:
+  name: dynamic-node <1>
+spec:
+  autoSizingReserved: true <2>
+  machineConfigPoolSelector:
+    matchLabels:
+      pools.operator.machineconfiguration.openshift.io/worker: "" <3>
+----
+<1> Assign a name to CR.
+<2> Add the `autoSizingReserved` parameter set to `true` to allow {product-title} to automatically determine and allocate the `system-reserved` resources on the nodes associated with the specified label. To disable automatic allocation on those nodes, set this parameter to `false`. 
+<3> Specify the label from the machine config pool.
++
+The previous example enables automatic resource allocation on all worker nodes. {product-title} drains the nodes, applies the kubelet config, and restarts the nodes.
+
+. Verify the `system-reserved` value:
+
+.. Log in to a node you configured:
++
+[source,terminal]
+----
+$ oc debug node/<node_name>
+----
+
+.. View details on the kubelet process:
++
+[source,terminal]
+----
+# ps -ef | grep kubelet
+----
++
+.Example output
+[source,terminal]
+----
+root        1613       1 11 06:49 ?        00:00:05 kubelet --config=/etc/kubernetes/kubelet.conf --bootstrap-kubeconfig=/etc/kubernetes/kubeconfig --kubeconfig=/var/lib/kubelet/kubeconfig --container-runtime=remote --container-runtime-endpoint=/var/run/crio/crio.sock --runtime-cgroups=/system.slice/crio.service --node-labels=node-role.kubernetes.io/worker,node.openshift.io/os_id=rhcos --node-ip= --minimum-container-ttl-duration=6m0s --volume-plugin-dir=/etc/kubernetes/kubelet-plugins/volume/exec --cloud-provider=azure --cloud-config=/etc/kubernetes/cloud.conf --pod-infra-container-image=quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:7b8e2e2857d8ac3499c9eb4e449cc3296409f1da21aa21d0140134d611e65b84 --system-reserved=cpu=0.07,memory=2.5Gi --v=2
+----
++
+In the previous example, the worker nodes are allocated `0.07` CPU and 2.5 Gi of memory. It can take several minutes for the optimal values to appear.
```

Comment:
Does it make sense to use `2.5` Gi?

Reply:
Those are the resources automatically assigned to the node by the cluster I grabbed the numbers from the [QE test case](https://polarion.engineering.redhat.com/polarion/#/project/OSE/workitems/testcase?query=trello%3AOCPNODE%5C-367) as I couldn't pull up a cluster-bot cluster.

Reply:
@mburke5678, oh, I meant does it need code font like for CPU?